kolibri-electron/src/index.js

```diff
@@ -0,0 +1,90 @@
+const { app, BrowserWindow } = require('electron');
+const path = require('path');
+const child_process = require('child_process');
+const http = require('http');
+
+// Handle creating/removing shortcuts on Windows when installing/uninstalling.
+if (require('electron-squirrel-startup')) { // eslint-disable-line global-require
+  app.quit();
+}
+
+const KOLIBRI = 'http://localhost:5000';
+
+const waitForKolibriUp = (splash, mainWindow) => {
+  console.log("Pinging kolibri server");
+  http.get(`${KOLIBRI}/static/assets/logo.ico`, (response) => {
```

Comment:
A better URL to poll might be...

```suggestion
  http.get(`${KOLIBRI}/api/public/info`, (response) => {
```

Static files might actually be on a different origin (and served by a different application) in a future version of Kolibri - see learningequality/kolibri#7920. So, polling a static file might not tell us what we expect. And that API provides a slightly smaller response along with some potentially useful information :)